### DISCUSSION TITLE
Regarding 'Game Launch'/'Bootup' Speeds

### DISCUSSION THREAD

**Discussion by rebornsunny2000** [skyline-emu/skyline#1731] [General]
So while I am using Yuzu and Ryujinx (and one time the other mobile emulator, you know the one), games take quite a while to launch, sometimes even around a minute. For example: Shin Megami Tensei V took around 47 seconds to boot up in Ryujinx, But in Skyline, all the games I've tested bootup within a few seconds, for example: Kill la Kill IF boots up in 3-4 seconds in Skyline while it takes around 27 seconds (for me at least) on the PC emulator. The other mobile emulator took a millenia for any game to bootup. So I just wanted to ask this question: Will Skyline also, in the future, take longer to launch games, since it will be using more stuff I think? or will the launch time stay relatively the same? The bootup speed for games is absolutely incredible here, so just wanted to get this out of my head. 
Thank you for any responses!

**Comment by ghost** [comment]
Probably not.
